### ISSUE TITLE
BK1 - Markup validation error

### ISSUE THREAD

**Issue by jpinsent** [ccswbs/hjckrrh#156]
[The Markup Validator](https://validator.w3.org/nu/?showsource=yes&useragent=Validator.nu%2FLV+http%3A%2F%2Fvalidator.w3.org%2Fservices&doc=https%3A%2F%2Fdev.web.uoguelph.ca%2Fuaic%2Facronyms-g) for Book detail pages gives the following error:
"Bad value up for attribute rel on element link: The string up is not a registered keyword."

**Comment by mmafe** [comment]
Issue caused by the Book Module: https://www.drupal.org/node/1911974

No clear solution immediately apparent, so moved out off hotfixes and into Next Release milestone.

**Comment by nevantan** [comment]
The "up" keyword for the "rel" attribute is not deprecated or obsolete, but it was "Dropped without Prejudice" from the HTML5 spec. 

This rel attribute is used by some browser extensions, but I don't believe this usage is extensive enough to justify the page failing validation. The link can safely be removed without breaking anything except the aforementioned extensions, but there are no issues/patches on the topic on drupal.org. This means we would likely have to write a patch for core ourselves to resolve the validation issue.

See: http://stackoverflow.com/questions/17415172/html-rel-up-attribute

**Comment by nevantan** [comment]
At @brharp's suggestion, it looks like overriding the template preprocess function may be a better option than patching the module. The relevant function is `template_preprocess_book_navigation` in `./modules/book/book.module:1092`.

**Comment by bdavey07** [comment]
http://donaldson-bdavey.c9users.io/iss156/book-page/test-book-page

**Comment by bdavey07** [comment]
https://dev.web.uoguelph.ca/uaic/acronyms-g
